dll/win32/advapi32/sec/lsa.c

```diff
@@ -29,9 +29,12 @@ LsapIsLocalComputer(PLSA_UNICODE_STRING ServerName)
 
     buf = HeapAlloc(GetProcessHeap(), 0, dwSize * sizeof(WCHAR));
     Result = GetComputerNameW(buf, &dwSize);
-    if (Result && (ServerName->Buffer[0] == '\\') && (ServerName->Buffer[1] == '\\'))
-        ServerName += 2;
-    Result = Result && !lstrcmpW(ServerName->Buffer, buf);
+    if (Result) {
+        if (ServerName->Buffer[0] == '\\' && ServerName->Buffer[1] == '\\')
+	    	Result = !lstrcmpW((LPCWSTR)&ServerName->Buffer[2], buf); 
+    	else
+	    	Result = !lstrcmpW(ServerName->Buffer, buf); 
+	} 
```

Comment:
Applied @binarymaster's suggestion + the TAB vs. 4-space indentation fix:
```suggestion
    if (Result)
    {
        PCWSTR pSrvName = ServerName->Buffer;
        if (pSrvName[0] == '\\' && pSrvName[1] == '\\')
            pSrvName += 2;
        Result = !lstrcmpW(pSrvName, buf); 
    } 
```

Reply:
I concur.
And remove the ending space on the `Result =` line.

Reply:
I've reduced the diff, now it should look more clear.

Reply:
I preferred it as Hermès suggested it: no extra parentheses, no redundant condition.

Reply:
For a change from first-time contributor the diff is recommended to be minimal. Refactoring could be done later.